finality-provider/config/config.go

```diff
@@ -26,17 +26,23 @@ const (
 	DefaultRPCPort                     = 12581
 	defaultConfigFileName              = "fpd.conf"
 	defaultNumPubRand                  = 50000 // support running of roughly 5 days with block production time as 10s
-	defaultNumPubRandMax               = 500000
-	defaultTimestampingDelayBlocks     = 6000 // 100 BTC blocks * 600s / 10s
-	defaultBatchSubmissionSize         = 1000
-	defaultRandomInterval              = 30 * time.Second
+	defaultBatchSubmissionSize         = 100
 	defaultSubmitRetryInterval         = 1 * time.Second
 	defaultSignatureSubmissionInterval = 1 * time.Second
 	defaultMaxSubmissionRetries        = 20
 	defaultBitcoinNetwork              = "signet"
 	defaultDataDirname                 = "data"
 )
 
+// Constants for system parameters validation limits
+const (
+	MaxBatchSize            = 100              // Maximum allowed batch submission size
+	MaxPubRand              = 500000           // Maximum allowed public randomness number
+	MinPubRand              = 8192             // Minimum allowed public randomness number
```

Comment:
Let's add comments explaining the reasoning behind the selection of the above? I'm not sure I understand the selections.